docs/tutorials/sending-value-programmatically.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: "Sending Value Programmatically | Interledger"
```

Comment:
Is there a plan for automatically generating the HTML from the markdown file? If not, it seems like it'll be annoying to try to keep the two in sync

Reply:
Yes, we are planning to.